client/src/shared/components/Text-field/index.tsx

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+
```

Comment:
- for shared file naming convention. please use _Pascal case_
![image](https://user-images.githubusercontent.com/73863685/223053348-51cba62e-7a72-42eb-abb5-d2a3ce043a01.png)